cfme/automate/buttons.py

```diff
@@ -16,6 +16,7 @@
 from widgetastic.xpath import quote
 from widgetastic_patternfly import BootstrapSelect, Button, CandidateNotFound, Input
 
+from cfme.base.ui import AutomateSimulationView
```

Comment:
Question : Any idea why is AutomateSimulationView in base.ui file ? There is simulation.py in automate folder . 
Not in this PR but I think it should be moved to respective automate classes

Reply:
@sshveta: I agreed with your point. But I try to avoid changes in others FAs without permission. As per my understanding it is old stuff. I will take a look on this thanks for pointing this.